components/date-picker/internal/DatePickerDatetimePanel.razor

```diff
@@ -71,23 +71,25 @@
             timeFormat = Format;
         }
 
+        var use12Hours = Use12Hours || timeFormat.ToLower().Contains("t");
+        var isPM = use12Hours && Value?.Hour >= 12;        
         DatePickerDisabledTime disabledTime = GetDisabledTime();
-		bool isValueNull = Value is null;
-		Func<int, int?, string> selected;
-		string localValue;
-		if (Value is null)
-		{
-			localValue = "";
-			selected = (_, _) => "";
-		}
-		else
-		{
-			localValue = Value.Value.ToString(timeFormat);
-			selected = (viewTime, valueTime) =>  viewTime == valueTime ? $"{PrefixCls}-time-panel-cell-selected" : "";
-		}		
+        bool isValueNull = Value is null;
+        Func<int, int?, string> selected;
+        string localValue;
+        if (Value is null)
+        {
+            localValue = "";
+            selected = (_, _) => "";
+        }
+        else
+        {
+            localValue = Value.Value.ToString(timeFormat);
+            selected = (viewTime, valueTime) => viewTime == valueTime ? $"{PrefixCls}-time-panel-cell-selected" : "";
+        }
```

Comment:
Please fix the indent.

Reply:
Ok. Do you mean it should be indented like below? 

![image](https://user-images.githubusercontent.com/3949302/172835781-6ce6382b-f843-4d5b-afb9-4bf94eec26da.png)

Reply:
Oh no, the razor files should be indent with two spaces, just follow the default setting of blazor.

Reply:
It was like this before I changed it. That's why I'm asking what indentation you mean.

Reply:
I'm not sure why but Visual Studio formats it like it was in my commit. I use ctrl+k+d and default settings.

Reply:
It looks like the standard indentation in VS 2022 is one TAB. It formats the lines you highlighted with two TABs to align with the previous code. The `<div class='@($"{PrefixCls}-datetime-panel")'` makes this indentation. Please check it in Visual Studio. It looks like the default formatting to me.

Reply:
Everything ok in vs diff 😀